From c699110b4b08e7fdee24d2b895403f15dad1502c Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Fri, 24 Feb 2012 12:08:43 -0500
Subject: [PATCH] libext2fs: include config.h for HAVE_xxx defines

We check HAVE_UNISTD_H but haven't included config.h yet, so we end up
hitting warnings about missing prototypes for close/read/etc... funcs.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 lib/ext2fs/mmp.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/lib/ext2fs/mmp.c b/lib/ext2fs/mmp.c
index b27d9a4..49a11da 100644
--- a/lib/ext2fs/mmp.c
+++ b/lib/ext2fs/mmp.c
@@ -13,6 +13,8 @@
 #define _GNU_SOURCE
 #endif
 
+#include "config.h"
+
 #if HAVE_UNISTD_H
 #include <unistd.h>
 #endif
-- 
1.7.8.4

